fvm/evm/handler/coa.go

```diff
@@ -0,0 +1,224 @@
+
+// COAContractABIJSON is the json string of ABI of the coa smart contract.
+var COAContractABIJSON = `
```

Comment:
Maybe consider refactoring the ABI JSON constants to files and use `go:embed` to integrate them

Reply:
updated to use embed for both json and bytecode.